[ja] Translate content/en/docs/reference/glossary/cloud-provider.md (#46417)
kubernetes/website: content/ja/docs/reference/glossary/cloud-provider.md

Reviewer: 全体を通してですが、"Cloud Provider" "Cloud Service Provider" という用語は固有名詞ではないので、カタカナ表記（クラウドプロバイダー / クラウドサービスプロバイダー）にした方が良いかなと思いました。

---

参考:

- Google Cloud: https://cloud.google.com/learn/what-is-a-cloud-service-provider?hl=ja
- Microsoft: https://azure.microsoft.com/ja-jp/resources/cloud-computing-dictionary/what-is-a-cloud-provider
- Red Hat: https://www.redhat.com/ja/topics/cloud-computing/what-are-cloud-providers

---
title: クラウドプロバイダー
id: cloud-provider
date: 2018-04-12
short_description: >

>   クラウドコンピューティングプラットフォームを提供する組織。

Reviewer 2: 下部と揃ってない
```suggestion
クラウドコンピューティングプラットフォームを提供する企業やその他の組織。
```

Reviewer: これは翻訳元の文章も異なっているからですね

aka:
  - クラウドサービスプロバイダー
tags:
  - community
---

クラウドコンピューティングプラットフォームを提供する企業やその他の組織。

<!--more-->

> クラウドプロバイダーは、クラウドサービスプロバイダー(CSP)と呼ばれることもあり、クラウドコンピューティングプラットフォームやサービスを提供しています。

Reviewer 2: ```suggestion
クラウドプロバイダーは、クラウドサービスプロバイダー(CSP)とも呼ばれ、クラウドコンピューティングプラットフォームやサービスを提供します。
```

> Cloud Providerは、Cloud Service Provider（CSP）と呼ばれることもあり、クラウドコンピューティングプラットフォームやサービスを提供しています。

Reviewer: ```suggestion
Cloud Providerは、Cloud Service Provider(CSP)と呼ばれることもあり、クラウドコンピューティングプラットフォームやサービスを提供しています。
```

nits: カッコは半角を用いていただくようお願いします

ref. https://kubernetes.io/ja/docs/contribute/localization/#basic-policy

> 記号類は感嘆符「！」と疑問符「？」のみ全角、それ以外は半角で表記

> 多くのCloud Providerは、マネージドインフラストラクチャー（Infrastructure as a ServiceやIaaSとも呼ばれる）を提供しています。

Reviewer: ```suggestion
多くのCloud Providerは、マネージドインフラストラクチャ(Infrastructure as a ServiceやIaaSとも呼ばれる)を提供しています。
```

"re" で終わる単語（infrastructure）には長音を付与しないようにお願いします

ref. https://kubernetes.io/ja/docs/contribute/localization/#long-vowel

> -ear、-eer、-re、-ty、-dy、-ryで終わる単語は長音を付与しない
>    - 例: 「クリア」「エンジニア」「アーキテクチャ」「セキュリティ」「スタディ」「ディレクトリ」

> マネージドインフラストラクチャでは、クラウドプロバイダーがサーバー、ストレージ、ネットワーキングの責任を負い、ユーザーはKubernetesクラスターの実行など、その上のレイヤーを管理します。

Reviewer 2: ```suggestion
マネージドインフラストラクチャでは、クラウドプロバイダーがサーバー、ストレージ、ネットワーキングの責任を負い、ユーザーはその上のレイヤー、例えばKubernetesクラスターの運用を管理します。
```

> マネージドインフラストラクチャーでは、Cloud Providerがサーバー、ストレージ、ネットワーキングの責任を負い、ユーザーはKubernetesクラスタの実行など、その上のレイヤーを管理します。

Reviewer: ```suggestion
マネージドインフラストラクチャでは、Cloud Providerがサーバー、ストレージ、ネットワーキングの責任を負い、ユーザーはKubernetesクラスターの実行など、その上のレイヤーを管理します。
```

"er" で終わる単語（cluster）には、長音を付与するようにお願いします

> - -er、-or、-ar、-cy、-gyで終わる単語は長音を付与する
>     - 例: 「クラスター」「セレクター」「サイドカー」「ポリシー」「トポロジー」

> Kubernetesはマネージドサービスとしても提供されています。これは、Platform as a Service(PaaS)と呼ばれることもあります。マネージドなKubernetesでは、クラウドプロバイダーは、Kubernetesのコントロールプレーンだけでなく、{{< glossary_tooltip term_id="node" text="Node" >}}と、それらが依存するインフラストラクチャ(ネットワーク、ストレージ、場合によってはロードバランサーなどの他要素)にも責任を負います。

Reviewer 2: ```suggestion
また、Kubernetesをマネージドサービスとして提供する場合もあります。これはPlatform as a Service(PaaS)とも呼ばれます。マネージドKubernetesでは、クラウドプロバイダーがKubernetesのコントロールプレーンや{{< glossary_tooltip term_id="node" text="Node" >}}、ネットワーキング、ストレージ、ロードバランサーなどのインフラストラクチャを管理します。
```

> Kubernetesはマネージドサービスとしても提供されています。これは、Platform as a Service（PaaS）と呼ばれることもあります。マネージドKubernetesでは、Cloud Providerは、Kubernetesのコントロールプレーンだけでなく、{{< glossary_tooltip term_id="node" text="ノード" >}}と、それらが依存するインフラストラクチャー（ネットワーク、ストレージ、場合によってはロードバランサーなどの他要素）にも責任を負います。

Reviewer: ```suggestion
Kubernetesはマネージドサービスとしても提供されています。これは、Platform as a Service(PaaS)と呼ばれることもあります。マネージドなKubernetesでは、Cloud Providerは、Kubernetesのコントロールプレーンだけでなく、{{< glossary_tooltip term_id="node" text="Node" >}}と、それらが依存するインフラストラクチャ(ネットワーク、ストレージ、場合によってはロードバランサーなどの他要素)にも責任を負います。
```
